NewForce-at-Mountwest/bangazon-site-take-2-fluffy
# Product types have a hyperlink

# Description

When the user is viewing any page on the bangazon site with a product type (category) it will be hyperlinked.

## Related Ticket(s)
12--https://trello.com/c/CXddT1b5/12-12-users-can-view-all-products-that-are-of-a-particular-type

## Steps to Test
--Add an commit on your own branch before checking out to sw-hyperlink-producttype
--Log in and run the app
--From the home page, or any page where you see a product category (for example "sporting goods") you should be able to click on the name of the category and go to the details page for that product type
-- you will see a list of all the products in that category with their title, quantity, and price.
--Places where you will see the product type:
* index view of products
* index view of product types
* details view of product
-- I have also added helpful links on the product and product types pages that allow you to view a list of all products, or a list of all product types.
--All other functionality should remain the same.


# Checklist:
